### DISCUSSION TITLE
How to delpoy flet app how to host the .py file on web

### DISCUSSION THREAD

**Discussion by plugeit** [flet-dev/flet#514] [Q&A] [answered]
I am new to flet and flet is also know so i can't find a step by step guide how to deploy flet .py file using firebase can someone help me to delpoy it i just need a video or written documentation

**Comment by ndonkoHenri** [comment]
https://flet.dev/docs/guides/python/deploying-web-app

**Comment by alwaysbetter1314** [comment]
As far as I know, google-firebase-hosting only supports deploying static html files and  Cloud Functions, however, flet uses server-side-render. Maybe you could deploy your app with fly.io.

**Reply by plugeit** [reply]
ok can you please elaborate

**Reply by ndonkoHenri** [reply, marked as answer]
https://flet.dev/docs/guides/python/deploying-web-app
